cmd/nfs-subdir-external-provisioner/provisioner.go

```diff
@@ -125,7 +129,7 @@ func (p *nfsProvisioner) Provision(ctx context.Context, options controller.Provi
 			AccessModes:                   options.PVC.Spec.AccessModes,
 			MountOptions:                  options.StorageClass.MountOptions,
 			Capacity: v1.ResourceList{
-				v1.ResourceName(v1.ResourceStorage): options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)],
+				v1.ResourceName(v1.ResourceStorage): options.PVC.Spec.Resources.Requests[(v1.ResourceStorage)],
```

Comment:
@humblec -- q_ why is this change required?

Reply:
keeping it intact for consistency... ptal. thanks.